experimental/Schemes/elliptic_surface.jl

```diff
@@ -93,8 +93,15 @@ end
 Return the generic fiber as an elliptic curve.
 """
 generic_fiber(S::EllipticSurface) = S.E
+
 weierstrass_chart(S::EllipticSurface) = S.Weierstrasschart
 
+@attr AbsSpec function weierstrass_chart_on_minimal_model(S::EllipticSurface)
```

Comment:
Just a minor side note: since `AbsSpec` is not a concrete type (it is missing type parameters, and it is abstract), this hint, while not completely useless, still won't make this accessor type stable. Perhaps there is no better type you can specify here, then so be it. Just wanted to make sure you are aware :-)

Reply:
Yes, I'm aware of that. Either way, this function is sufficiently high-level that type stability shouldn't be an issue of speed. The type assertion is just here as a plausibility check and to improve readability of the code.